src/app/code/community/Fontis/Australia/Model/Address/LocalLookup.php

```diff
@@ -86,11 +86,10 @@ public function getPostcodeValidationResults($state, $suburb, $postcode, $countr
             $result = $conn->fetchAll(
                 'SELECT DISTINCT au.*, dcr.region_id FROM ' . $res->getTableName('australia_postcode') . ' AS au
              RIGHT JOIN ' . $res->getTableName('directory_country_region') . ' AS dcr ON au.region_code = dcr.code
-             WHERE (postcode = :postcode)  
-             OR (city = :city) 
-             OR (city = :city AND postcode = :postcode) 
-             OR (postcode = :postcode AND region_code = :state) 
-             AND dcr.country_id = :country_id
```

Comment:
Even though this `and` statement filters by country id, washington was still being joined for the other `or` statements. Not sure if this is a clean way to solve it, but seems to do the job